script.js

```diff
@@ -0,0 +1,90 @@
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Itaque tenetur quaerat repellat sint doloribus numquam hic!Fugiat illo at eligendi a nihil ullam eos voluptatem nesciunt?',
+  },
+  {
+    image1: 'images/chess.jpg',
+    image2: 'images/harris.jpg',
+    name: 'Harris',
+    headline: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit.',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Itaque tenetur quaerat repellat sint doloribus numquam hic!Fugiat illo at eligendi a nihil ullam eos voluptatem nesciunt?',
+  },
+  {
+    image1: 'images/chess.jpg',
+    image2: 'images/zeeshan.jpg',
+    name: 'Zeeshan',
+    headline: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit.',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Itaque tenetur quaerat repellat sint doloribus numquam hic!Fugiat illo at eligendi a nihil ullam eos voluptatem nesciunt?',
+  },
+  {
+    image1: 'images/chess.jpg',
+    image2: 'images/pachal.jpg',
+    name: 'Hamza',
+    headline: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit.',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Itaque tenetur quaerat repellat sint doloribus numquam hic!Fugiat illo at eligendi a nihil ullam eos voluptatem nesciunt?',
+  },
+  {
+    image1: 'images/chess.jpg',
+    image2: 'images/zain.jpg',
+    name: 'Zain',
+    headline: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit.',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Itaque tenetur quaerat repellat sint doloribus numquam hic!Fugiat illo at eligendi a nihil ullam eos voluptatem nesciunt?',
+  },
+  {
+    image1: 'images/chess.jpg',
+    image2: 'images/hamza.jpg',
+    name: 'Hamza',
+    headline: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit.',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Itaque tenetur quaerat repellat sint doloribus numquam hic!Fugiat illo at eligendi a nihil ullam eos voluptatem nesciunt?',
+  },
+];
+
+/*  eslint-disable no-return-assign */
+// eslint-disable-next-line no-return-assign
+
+if (cardSection) {
+  cardsObj.map(
+    (card) => (cardSection.innerHTML += `<div class="col-6"><div class="card spekrCard"> <div class="spekrImg"> <img src="${card.image1}" alt="chess" class="card-img-left spekrImg1" /> <img src="${card.image2}" alt="Speaker's Image" class="card-img-left spekrImg2" /> </div> <div class="card-body"> <div class="card-title spekrTitle">${card.name}</div> <p class="card-text spekrHeading"> ${card.headline} </p> <p class="card-text text-black spekrText"> ${card.desciption} </p> </div> </div> </div>`),
+  );
+}
+
+const btn1 = document.querySelector('.main_btn');
+
+function on() {
+  document.getElementById('popup').style.display = 'block';
+  document.body.style.overflowY = 'hidden';
+}
+
+if (btn1) {
+  btn1.addEventListener('click', on);
+}
```

Comment:
- [ OPTIONAL ] I think here you should add a gap since the text here is very close to the image and in the original design there a space so please add a gap to make your design identical to the original design as much as possibel 💯🔥♻

- [ OPTIONAL ] I think here the color of the first heading not the same as the design so kindly change it to be the same ✅

- Your design
![image](https://user-images.githubusercontent.com/58553711/205466375-d0292c9b-33be-4156-9aa5-73ab860459fb.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/205466407-b8c8796d-50c6-4d47-8796-277f30fafc3a.png)

Reply:
Thanks for the suggestion. It is working fine in full secreen